main.js

```diff
@@ -1,24 +1,18 @@
 const fs = require('fs');
 const discord = require('discord.js');
+
 const DISCORD_MAX_DESCRIPTION_LENGTH = 2048;
 const DISCORD_GUILD = '238666723824238602';
 const DISCORD_CHANNEL = '308772291863642112';
 const ORDERED_LIST_REGEX = new RegExp('[0-9]+.(.*)');
+const DISCORD_TOKEN = process.env.DISCORD_TOKEN;
 
-const tokenFactory = () => process.env.DISCORD_TOKEN;
-const DISCORD_TOKEN = tokenFactory();
-
-const getMarkdownText = async () => (
-    new Promise((res, rej) => fs.readFile('README.md', 'utf8', (err, data) => {
-        if (err) return rej(err);
-        res(data);
-    }))
-);
+const getMarkdownText = () => fs.readFileSync('README.md', 'utf8');
```

Comment:
We can't do anything without reading this so, why not make it synchronous?

Reply:
No particular reason, I was fascinated by async apis at the time I suppose.